src/styles/stars.css

```diff
@@ -0,0 +1,31 @@
+ul.stars {
+  margin: 0;
+}
```

Comment:
@naziamoff that's a good question. Ihor told me to use normalize.css. If I add ul { margin:0; } to stars.css would it be correct for BEM? Because the solution with resetting  ul margins inside stars.css sounds pretty good))